### PR TITLE
🐛 PLAT-259 Set mime type on file

### PR DESCRIPTION
**Pull request by dmihalcik-virtru**
PLAT-259: Use the mimeType field in the tdf3 spec

### CODE REVIEW

**Review by pflynn-virtru — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "react-dom": "^16.13.0",
     "react-router-dom": "^5.1.2",
     "redux-zero": "^5.1.3",
-    "virtru-sdk-current": "npm:virtru-sdk@^2.0.1",
-    "virtru-sdk-lts": "npm:virtru-sdk@^1.6.16"
+    "virtru-sdk-current": "npm:virtru-sdk@^2.1.0-beta.19",
```
Comment (pflynn-virtru) [inline package.json]: build failing: "Please update your lock file with `npm install` before continuing."

**Review by pflynn-virtru — DISMISSED** [review]
I updated package-lock.json

npm audit fails now (unrelated, for most projects)

**Review by pflynn-virtru — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (jrschumacher) [reply package.json:19]: Woah... I've never seen this before. What does prepending with `npm:` do?

**Review by jrschumacher — APPROVED** [review]
LGTM

**Pull request merged**